sunpy_soar/client.py

```diff
@@ -170,12 +171,17 @@ def _do_search(query):
         r = requests.get(f"{tap_endpoint}/sync", params=payload)
         log.debug(f"Sent query: {r.url}")
         r.raise_for_status()
+        try:
+            response_json = r.json()
+        except JSONDecodeError as err:
+            msg = "Server returned an invalid JSON response. The SOAR server may be down or not functioning correctly."
```

Comment:
```suggestion
            msg = (
                "The SOAR server returned an invalid JSON response. "
                "It may be down or not functioning correctly."
            )
```
Not sure if the python whitespace is correct.

Reply:
The test match will need to be updated.